sn_cli/Cargo.toml

```
@@ -28,7 +28,7 @@ dirs-next = "2.0.0"
 ed25519-dalek = { version = "1.0.1", features = ["serde"] }
 hex = "~0.4"
 human-panic = "1.0.3"
-isatty = "~0.1"
+atty = "0.2.14"
```

```suggestion
atty = "~0.2.14"
```
Because it's not yet a major version, we need to add `~` to the version number to ensure we get patch updates

---

Ah! I see, pushing the change.